frontend/packages/ceph-storage-plugin/src/constants/providers.ts

```diff
@@ -16,6 +16,20 @@ export const PROVIDERS_NOOBAA_MAP = {
   [BC_PROVIDERS.IBM]: 'ibmCos' as const,
 };
 
+export const NS_PROVIDERS_NOOBAA_MAP = {
+  [BC_PROVIDERS.AWS]: 'awsS3' as const,
```

Comment:
why type assertion here ?

Reply:
this file got changed accidentally . these changes are already in as part of previous PR